examples/spec/ocr2-bootstrap.spec.toml

```diff
@@ -14,6 +14,7 @@ ocr2ProgramID      = "<insert solana ocr2 program ID>"
 transmissionsID    = "<insert solana ocr2 transmissions account>"
 storeProgramID     = "<insert solana ocr2 store account>"
 commitment         = "confirmed" # optional, defaults to "confirmed"
+txTimeout          = "1m"        # optional, defaults to "1m"
```

Comment:
Do you mind using this PR to just add human-readable descriptions to a README file, or some other `.md` file under `./docs` about the specific purpose of these knobs? 

Something like Connor did for Terra in a Notion doc describing NodeOps setup, but we'll also move those docs to the Terra repo as a single source of truth.

Reply:
can do! i'll add a few more items to my todo list at the top